blocked-edges/4.16.53-RuncShareProcessNamespace.yaml

```diff
@@ -0,0 +1,7 @@
+to: 4.16.53
+from: .*
+url: https://issues.redhat.com/browse/RUN-3748
+name: RuncShareProcessNamespace
+message: Some runc 1.2 releases fail to launch containers in Pods where shareProcessNamespace is explicitly set true.
```

Comment:
and `hostNetwork: true`

Reply:
It seems like we're still scoping exposure, so I've pushed f16e5d1e -> 7db3904f to pivot to the less specific `Some runc releases fail to launch containers in some Pods.`, and we can push refinements in follow-up pulls as we clarify our understanding.

/hold cancel